patches/eslint-plugin-eslint-plugin+3.5.3.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/node_modules/eslint-plugin-eslint-plugin/lib/utils.js b/node_modules/eslint-plugin-eslint-plugin/lib/utils.js
+index e00aada..555ec69 100644
+--- a/node_modules/eslint-plugin-eslint-plugin/lib/utils.js
++++ b/node_modules/eslint-plugin-eslint-plugin/lib/utils.js
+@@ -76,10 +76,10 @@ function isNormalFunctionExpressionReference (node, scopeManager) {
+ */
+ function isRuleTesterConstruction (node) {
+   return node.type === 'NewExpression' && (
+-    (node.callee.type === 'Identifier' && node.callee.name === 'RuleTester') ||
++    (node.callee.type === 'Identifier' && node.callee.name === 'GraphQLRuleTester') ||
```

Comment:
we need to patch `eslint-plugin-eslint-plugin` because our `ruleTester` is named differently